src/pybind/mgr/cephadm/module.py

```diff
@@ -1795,6 +1795,34 @@ def exit_host_maintenance(self, hostname: str) -> str:
 
         return f"Ceph cluster {self._cluster_fsid} on {hostname} has exited maintenance mode"
 
+    @handle_orch_error
+    @host_exists()
+    def rescan_host(self, hostname: str) -> str:
+        """Use cephadm to issue a disk rescan on every each HBA
+
+        Some HBA and external enclosures don't automatically register
+        device insert with the kernel, so for these scenarios we need
+        to manually recan
+
+        :param hostname: (str) host name
+        """
+        self.log.info(f'disk rescan request sent to host "{hostname}"')
+        _out, _err, _code = self.wait_async(CephadmServe(self)._run_cephadm(hostname, cephadmNoImage, "disk-rescan",
+                                                                            [],
+                                                                            no_fsid=True,
```

Comment:
Nit: is it too late to use ‘drive’ instead of ‘disk’?

Reply:
From my perspective,yes. However, if this gets merged feel free to raise a subsequent PR